source/fem-dem/particle_projector.cc

```diff
@@ -1296,8 +1295,8 @@ ParticleProjector<dim>::calculate_field_projection(
 
                   if (total_volume_of_particle_in_sphere > 0)
                     {
-                      local_rhs(i) += phi_vf[i] * particle_field_in_sphere *
-                                      fe_values_field.JxW(q);
+                      local_rhs(i) += phi_vf[i] * particle_field_in_sphere; // *
+                      //    fe_values_field.JxW(q);
                     }
```

Comment:
This I need to clean up. When it's distirbute it should be like this, otherwise it should be with the JxW